### ISSUE TITLE
Move Concrete Implementations to Implementation-Specific Repositories

### ISSUE THREAD

**Issue by VaughnVernon** [vlingo/vlingo-symbio#3]
The DynamoDB implementation is very heavyweight because its use requires the download of the local test environment dependencies for DynamoDB. On a non-optimal network that I had to use recently it required approximately 3 hours to get all the dependencies. (It's attributed to how Amazon has packaged the dependencies, which requires downloading all revisions and patches to the Java API that ever existed.) Although not a fault of the author/contributor, this is unacceptable and will hamstring adoption of vlingo-symbio for those not interested in a DynamoDB implementation. Moving this outside the vlingo-symbio core also sets a precedent for how other implementations will/can be provided.

-- @kmruiz Please move the DynamoDB support to vlingo-symbio-dynamodb
-- I will move the JDBC support to vlingo-symbio-jdbc

**Comment by kmruiz** [comment]
I think it's a really good idea, I'll move the DynamoDB support to another repository ASAP.

However, correct me if I'm wrong, what delayed to download are the test dependencies for DynamoDB (DynamoDB local itself IIRC) so it should not affect the actual developer using vlingo-symbio no?

**Comment by VaughnVernon** [comment]
True, but if someone wants to add CouchDB support, for example, they would have to wait on the DynamoDB dependencies before they could implement and test. That's not a welcome I would want from my first contribution (or any!) with vlingo/platform contribution. Of course if someone wants to make some specific vlingo-symbio-dynamodb contributions, the downloads are a necessary cost.

BTW, I have already done the port. I have zipped up the files and will email them to you so you can do the commit, etc.

**Comment by kmruiz** [comment]
Working on them, just checking that everything is fine and I will do the PR today if everything worked 😄

**Comment by kmruiz** [comment]
We already moved it, can this be closed? Or do we need to track something else?

Thanks!

**Comment by VaughnVernon** [comment]
Oops, closed now.
